addons/point_of_sale/models/pos_session.py

```diff
@@ -155,31 +155,37 @@ def _load_pos_data(self, data):
         data[0]['_has_cash_move_perm'] = self.env.user.has_group('account.group_account_invoice')
         data[0]['_has_available_products'] = self._pos_has_valid_product()
         data[0]['_pos_special_products_ids'] = self.env['pos.config']._get_special_products().ids
-        return {
-            'data': data,
-            'fields': fields
-        }
+        return data
 
-    def load_data(self, models_to_load, only_data=False):
+    def load_data(self, models_to_load):
         response = {}
         response['pos.session'] = self._load_pos_data(response)
-        self._load_pos_data_relations('pos.session', response)
 
         for model in self._load_pos_data_models(self.config_id.id):
             if models_to_load and model not in models_to_load:
                 continue
 
             try:
                 response[model] = self.env[model]._load_pos_data(response)
-            except AccessError as e:
-                response[model] = {
-                    'data': [],
-                    'fields': self.env[model]._load_pos_data_fields(response['pos.config']['data'][0]['id']),
-                    'error': e.args[0]
-                }
+            except AttributeError:  # FIXME AttributeError isn't the best exception to catch here
```

Comment:
> # FIXME AttributeError isn't the best exception to catch here

Why? Maybe put a reason so that the future person will have more context.

Reply:
Its for me ;)